src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -208,4 +208,16 @@ protected function getContainerExtensionClass()
 
         return $this->getNamespace().'\\DependencyInjection\\'.$basename.'Extension';
     }
+
+    /**
+     * Creates the bundle's container extension.
+     *
+     * @return ExtensionInterface|null
+     */
+    protected function createContainerExtension()
+    {
+        if (class_exists($class = $this->getContainerExtension())) {
```

Comment:
Looks wrong to me, should be getContainerExtensionClass, no? Can you add some tests as well?